pkg/commands/process/settings/rules/ruby/lang/exception.yml

```diff
@@ -0,0 +1,43 @@
+type: "risk"
+patterns:
+  - pattern: |
+      raise($<DATA_TYPE>)
+    filters:
+      - variable: DATA_TYPE
+        detection: datatype
+languages:
+  - ruby
+trigger: "local"
+severity:
+  default: "low"
+  PII: "critical"
+  PHI: "medium"
+  PD: "high"
+skip_data_types:
+  - "Unique Identifier"
+metadata:
+  description: "Do not send sensitive data to exception."
+  remediation_message: |
+    ## Description
+
+    Leaking sensitive data to exception is a common cause of data leaks and can lead to data breaches. This rule looks for instances of sensitive data sent to exceptions.
+
+    ## Remediations
+
+    ❌ Avoid using sensitive data in logger messages:
+
+    ```ruby
+    raise CustomException.new(user.email)
+    ```
+
+    ✅ If you need to identify a user, ensure to use their unique identifier instead of their personal identifiable information:
+
+    ```ruby
+    CustomException.new(user.uuid)
```

Comment:
should this be `raise CustomException...`?

Reply:
```suggestion
    raise CustomException.new(user.uuid)
```